instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpClientAttributesGetter.java

```diff
@@ -21,7 +22,8 @@
 public interface HttpClientAttributesGetter<REQUEST, RESPONSE>
     extends HttpCommonAttributesGetter<REQUEST, RESPONSE>,
         NetworkAttributesGetter<REQUEST, RESPONSE>,
-        ServerAttributesGetter<REQUEST> {
+        ServerAttributesGetter<REQUEST>,
+        UrlAttributesGetter<REQUEST> {
```

Comment:
`UrlAttributesGetter` is for server instrumentations. Http client semantic conventions https://github.com/open-telemetry/semantic-conventions/blob/5816d4bb099b69eba1cb45084d7cceb51ad6bbfe/docs/http/http-spans.md#http-client don't list all of the attributes that `UrlAttributesGetter` provides.

Reply:
could I add an interface like
```java
public interface HttpClientExperimentalAttributesGetter<REQUEST, RESPONSE> extends
    HttpClientAttributesGetter<REQUEST, RESPONSE> {}
```
and change
```java
  public static <REQUEST, RESPONSE> AttributesExtractor<REQUEST, RESPONSE> create(
      HttpClientAttributesGetter<REQUEST, RESPONSE> getter) {
    return new HttpExperimentalAttributesExtractor<>(getter);
  }
```
to take in a `HttpClientExperimentalAttributesGetter`?